test/integration/029_docs_generate_tests/test_docs_generate.py

```diff
@@ -1996,20 +1996,20 @@ def expected_postgres_references_manifest(self, model_database=None):
             	'exposure.test.notebook_exposure': {
                     'depends_on': {
                         'macros': [],
-                        'nodes': ['model.test.model', 'model.test.second_model']
+                        'nodes': ['model.test.view_summary']
                     },
                     'description': 'A description of the complex exposure',
                     'fqn': ['test', 'notebook_exposure'],
                     'maturity': 'medium',
                     'name': 'notebook_exposure',
-                    'original_file_path': self.dir('models/schema.yml'),
+                    'original_file_path': 'models/schema.yml',
```

Comment:
The test rendered this as ref_models/schema.yml so I'm removing this on the assumption that it causes it

Reply:
🤦 There's a folder called `ref_models`.